package.json

```diff
@@ -116,11 +118,13 @@
     "@metamask/obs-store": "^5.0.0",
     "@metamask/post-message-stream": "^4.0.0",
     "@metamask/providers": "^8.1.1",
+    "@ngraveio/bc-ur": "^1.1.6",
     "@popperjs/core": "^2.4.0",
     "@reduxjs/toolkit": "^1.6.2",
     "@sentry/browser": "^6.0.0",
     "@sentry/integrations": "^6.0.0",
-    "@zxing/library": "^0.8.0",
+    "@zxing/browser": "^0.0.10",
+    "@zxing/library": "0.8.0",
```

Comment:
why the exact dep?

Reply:
```suggestion
    "@zxing/library": "^0.8.0",
```

Reply:
Because the latest zxing library have some issues and limitation for our usage, so we lock the version here. @soralit can share more context about it.

Reply:
The `@zxing/library` we used before didn't support continuously scanning, which is required for animated QR codes. So we add `@zxing/browser` to use the latest feature. However, after importing the package, I found the `@zxing/library` is auto updated to `0.8.5` whose `BrowserQRCodeReader` was deprecated and will make the `qr-scanner` component not working.

Reply:
im concerned about depending on a deprecated component

Reply:
We have planned an refactor to the QR scanning part of Metamask, not just our function but all the QR Code related code in Metamask. But we think it is better to do it in a separate pr about it which can be more focused. @kumavis